transactions/transactions_service.go

```diff
@@ -263,13 +265,17 @@ func (svc *transactionsService) SendKeysend(ctx context.Context, amount uint64,
 	metadata := map[string]interface{}{}
 
 	metadata["destination"] = destination
-	metadata["tlv_records"] = customRecords
-	metadataBytes, err := json.Marshal(metadata)
 
+	// we copy records because parseMetadata decodes record values
```

Comment:
Is this actually needed? and I think it would be good for `parseMetadata` to not do this anyway so there is no side effect

Reply:
Done!